### PR TITLE
Color picker interaction tests

### PR DESCRIPTION
**Pull request by laviomri**

### CODE REVIEW

File: src/__tests__/interactions-helper.js [hunk 1]
```diff
@@ -108,6 +108,11 @@ export const waitForElementVisible = getterFunc => {
   });
 };
 
+export const keyboardMultipleTimes = async (text, times, options = { delay: 70 }) => {
```
Comment (hadasfa) [inline src/__tests__/interactions-helper.js:111]: is the word time is the correct word? it's not about time it's about amount (can be confiusing)

File: src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ export const ColorPickerColorsGrid = React.forwardRef(
               colorSize={colorSize}
               tooltipContent={tooltipContentByColor[color]}
               colorShape={colorShape}
+              data-testid={`color-picker-item-${color}`}
```
Comment (hadasfa) [inline src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx:58]: I saw we create get test id function, should we use it? it create a data test id which depend on element type enum and also the id

File: src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx [hunk 3]
```diff
@@ -62,6 +63,7 @@ const ColorPickerItemComponent = ({
           active: isActive,
           circle: colorShape === COLOR_SHAPES.CIRCLE
         })}
+        data-testid={dataTestId}
```
Comment (hadasfa) [inline src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx:66]: same question about data test id

File: src/__tests__/interactions-helper.js (hunk 1 above)
Comment (laviomri) [reply src/__tests__/interactions-helper.js:111]: will update to "count" 👍

**Review by hadasfa — APPROVED** [review]

**Pull request merged**